src/main/java/uk/gov/pay/api/exception/CreateChargeConnectorErrorResponseExceptionMapper.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.pay.api.exception;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.pay.api.model.PaymentErrorResponse;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+import static javax.ws.rs.core.Response.Status.NOT_FOUND;
+import static uk.gov.pay.api.exception.CreateChargeConnectorErrorResponseExceptionMapper.CreateChargeErrorResponse.P0198;
+import static uk.gov.pay.api.exception.CreateChargeConnectorErrorResponseExceptionMapper.CreateChargeErrorResponse.P0199;
+
+public class CreateChargeConnectorErrorResponseExceptionMapper implements ExceptionMapper<CreateChargeConnectorErrorResponseException> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CreateChargeConnectorErrorResponseException.class);
+
+    @Override
+    public Response toResponse(CreateChargeConnectorErrorResponseException exception) {
+
+        int errorStatus = exception.getErrorStatus();
+
+        if (errorStatus == NOT_FOUND.getStatusCode()) {
+            LOGGER.warn("Authorization succeeded but Gateway account was not found in Connector", errorStatus);
+            return P0199.response();
```

Comment:
is logger.warn good enough for this, I see this as a rare and severe exception to deal with, should be error, or even fatal